lua/nordic/colors/init.lua

```diff
@@ -3,13 +3,15 @@ local P = require 'nordic.colors.nordic'
 
 local C = {}
 
-local function build_palette()
-    -- Make a new palette.
-    -- Copy the original palette this overrides C.
-    C = vim.deepcopy(P)
-    -- Because of the override we need to re-add the build_palette to the C table.
-    C.build_palette = build_palette
-    -- If we had attached build_palette to C we would lost it after the override.
+function C.build_palette()
+    -- clear all values except build_palette
+    for k, _ in pairs(C) do
+        if k ~= "build_palette" then
+            C[k] = nil
+        end
+    end
```

Comment:
I can't remember, but why do we have to clear all of the values before merging?  Can't we just merge in place directly?

Reply:
I don't remember either:
[refactor(colors): clean up build_palette;](https://github.com/AlexvZyl/nordic.nvim/pull/137/commits/4c697537d020fa2df54c89e3d04f9ad8cc542579)